dashboard/test/ui/features/learning_platform/lesson_lock.feature

```diff
@@ -48,10 +48,10 @@ Scenario: Lock settings for students
   And I wait until element "td:contains(Anonymous student survey 2)" is visible
   And I wait until jQuery Ajax requests are finished
   Then element "td:contains(Anonymous student survey 2) .fa-unlock" is visible
-  Then I verify progress for lesson 31 level 1 is "not_tried"
-  Then I verify progress for lesson 31 level 2 is "not_tried"
-  Then I verify progress for lesson 31 level 3 is "not_tried"
-  Then I verify progress for lesson 31 level 4 is "not_tried"
+  Then I verify progress for lesson 31 level 1 is "not_tried" without waiting
```

Comment:
The tests in this file verify "not_tried" several times in a row which can be inefficient so I decided to keep the test as-is, waiting once (e.g. on line 49) and then verifying without waiting.